test/libsolidity/syntaxTests/inheritance/override/common_base_and_unique_implementation.sol

```diff
@@ -0,0 +1,14 @@
+interface I {
+    function f() external;
+    function g() external;
+}
+abstract contract A is I {
+    function f() external override {}
+    function g() external override virtual;
+}
+abstract contract B is I {
+    function g() external override {}
+    function f() external override virtual;
+}
+contract C is A, B {
```

Comment:
I see no reason to disallow this one, but I'm not sure you agree @chriseth?

Reply:
Well... One could argue that the only reason to mention the unimplemented ones in ``A`` and ``B`` again would be to prevent exactly this situation... so maybe we actually *should* require an explicit override in ``C`` here...